input/pagecontent/change_log.md

```diff
@@ -1,3 +1,62 @@
+**The following changes occurred between [STU 3 publication](https://hl7.org/fhir/us/mcode/STU3/) (October 2023) and the STU 4 ballot (May 2024). For a history of previous changes, please see the prior change logs in the [appropriate versions](http://hl7.org/fhir/us/mcode/history.html).**
+
+### Addition of non-TNM Staging Profiles
+
+mCODE 4.0.0-ballot includes additional non-TNM staging profiles. These staging profiles are especially important for pediatric oncology.
+
+* [NeuroblastomaINSStage]
+* [NeuroblastomaINRGSSRiskAssessment]
+* [WilmsTumorStage]
+* [RhabdomyosarcomaClinicalGroupStage]
+
+### Modification of Lymphoma Stage Profile
+
+Lymphoma staging can include many modifiers, such as bulky status, extranodal involvement, splenic involvement, and symptom indicators. As a result, components for these modifiers were added to the [LymphomaStage] profile to better describe the lymphoma stage.
+
+### Addition of Risk Assessment Profiles
+
+mCODE STU3 did not include any profiles for risk assessments, so there was no indication on how to use mCODE to represent different risk assessments. To address this issue, an abstract, generic profile was created for risk assessment, and specific risk assessment profiles were added for illustrative purposes:
```

Comment:
I wouldn't say this is a deficit of STU3, but rather that Risk Assessments were recognized as essential for representing the case data for a Cancer Patient.